examples/plugins/http-requests/http-requests.py

```diff
@@ -120,21 +118,21 @@ def do_report(self):
                         'priority': 0.1,
                     }
                 }
-            }
+            },
+            'Plugins': [
+              {
+                'id': 'http-requests',
+                'label': 'HTTP Requests',
+                'description': 'Adds http request metrics to processes',
+                'interfaces': ['reporter'],
+                'api_version': '1',
+              }
+            ]
         }
         body = json.dumps(report)
         self.request_log = "resp_size=%d, resp_entry_count=%d" % (len(body), len(process_nodes))
         self.respond(body)
 
-    def do_handshake(self):
-        spec = {
-            'name': 'http-requests',
-            'description': 'Adds http request metrics to processes',
-            'interfaces': ['reporter'],
-            'api_version': '1',
-        }
-        self.respond(json.dumps(spec))
-
     def respond(self, body):
```

Comment:
We should inline this (and maybe simplify the request_log handling since there's only one request type)